setup.py

```diff
@@ -26,6 +26,8 @@
     'prompt_toolkit>=3.0.6,<4.0.0',
     'PyMySQL >= 0.9.2',
     'sqlparse>=0.3.0,<0.5.0',
+    'sqlglot>=5.1.3',
+    'python-dateutil>=2.8.2',
```

Comment:
What is the need for the `python-dateutil` package?

Reply:
sqlglot does not require it, although it's an option dependency for the optimizer. it's not really needed for a cli tool

Reply:
Thanks, removed.